go.mod

```diff
@@ -38,6 +38,8 @@ require (
 	gopkg.in/urfave/cli.v1 v1.20.0
 )
 
+replace github.com/unicornultrafoundation/go-u2u/crypto/secp256k1 => github.com/ethereum/go-ethereum/crypto/secp256k1 v1.10.26
+
```

Comment:
Should bring the Geth `secp256k1` lib to our core instead of using Geth as upstream. This crypto lib won't be changed much in the future, and we also want to gather all Geth lib into this monorepo

Reply:
Alternatively, we could place it in a separate repository. The main repository has become quite large and might be overwhelming for beginners.